src/Hooks/useCheckFriendStatus.tsx

```diff
@@ -0,0 +1,15 @@
+import { useState,useEffect } from "react";
```

Comment:
Please correct filename.
Follow proper naming convention and be consistent in it at all places.